macros/schema_tests/table_shape/expect_table_column_count_to_equal.sql

```diff
@@ -1,6 +1,8 @@
 {%- macro test_expect_table_column_count_to_equal(model, value) -%}
 {%- if execute -%}
 {%- set number_actual_columns = (adapter.get_columns_in_relation(model) | length) -%}
-select {{ (number_actual_columns - value) | abs }}
+select 1
+from (select 1) a
+where {{ number_actual_columns }} != {{ value }}
```

Comment:
This one, and the ones below it, feel tricky—especially to find the right boundary between Jinja calculation and SQL calculation. I could see wanting to return a row that contains two columns, `number_actual_columns` and `value`, and then set `fail_calc` to be their difference.
```sql
{{ config(fail_calc = "abs(number_actual_columns - value)") }}
select {{ number_actual_columns }} as number_actual_columns, {{ value }} as value
```
That way the debug query / stored-in-database table is just slightly more informative. What do you think?

Reply:
Thanks @jtcohen6! I took some of these suggestions into a new branch!